subprojects/ivy/src/integTest/groovy/org/gradle/api/publish/ivy/IvyPublishHttpIntegTest.groovy

```diff
@@ -140,12 +135,8 @@ credentials {
     def "can publish to authenticated repository using #authScheme auth"() {
         given:
         org.gradle.api.credentials.PasswordCredentials credentials = new DefaultPasswordCredentials('testuser', 'password')
-        buildFile << publicationBuild(version, group, ivyHttpRepo.uri, "ivy", """
```

Comment:
❓ Won't we be losing important test coverage for inline credentials that way?

Reply:
Could add another variant for inlined credentials but disable for CC:

```
    @UnsupportedWithConfigurationCache(because = "inline credentials")
    def "can publish to authenticated repository using inline credentials and #authScheme auth"() {
        given:
        PasswordCredentials credentials = new DefaultPasswordCredentials('testuser', 'password')
        buildFile << publicationBuild(version, group, ivyHttpRepo.uri, "ivy","""
        credentials(PasswordCredentials) {
            username = "${credentials.username}"
            password = "${credentials.password}"
        }
        """)

        and:
        server.authenticationScheme = authScheme
        expectPublishModuleWithCredentials(module, credentials)

        when:
        run 'publish'

        then:
        module.assertMetadataAndJarFilePublished()
        module.jarFile.assertIsCopyOf(file('build/libs/publish-2.jar'))

        and:
        progressLogging.uploadProgressLogged(module.ivy.uri)
        progressLogging.uploadProgressLogged(module.moduleMetadata.uri)
        progressLogging.uploadProgressLogged(module.jar.uri)

        where:
        authScheme << [AuthScheme.BASIC, AuthScheme.DIGEST, AuthScheme.NTLM]
    }
```

Reply:
Yeah, let's do that 👍 .